server/api.go

```diff
@@ -235,7 +235,15 @@ func (p *Plugin) handleEndCall(w http.ResponseWriter, r *http.Request, channelID
 			return
 		}
 
-		p.LogInfo("call state is still in store, force ending it")
+		p.LogInfo("call state is still in store, force ending it", "channelID", channelID)
+
+		if state.Call.Recording != nil && state.Call.Recording.EndAt == 0 {
+			p.LogInfo("recording is in progress, force ending it", "jobID", state.Call.Recording.JobID)
```

Comment:
We may want to log the channelID here (and in the other log statements added below), to make it simpler?

Reply:
It certainly won't hurt. Updated.